assessment/models.py

```diff
@@ -462,6 +461,26 @@ class TestFlowAttempt(models.Model):
     event_participation = models.ForeignKey('assessment.AssessmentEventParticipation', on_delete=models.CASCADE)
     test_flow_attempted = models.ForeignKey('assessment.TestFlow', on_delete=models.RESTRICT)
 
+class ResponseTest(AssessmentTool):
+    sender = models.ForeignKey('users.Assessor', on_delete=models.CASCADE)
+    subject = models.TextField(null=False)
+    prompt = models.TextField(null=False)
+
+class ResponseTestSerializer(serializers.ModelSerializer):
+    owning_company_name = serializers.ReadOnlyField(source=OWNING_COMPANY_COMPANY_NAME)
+    sender = serializers.ReadOnlyField(source='sender.email')
+    class Meta:
+        model = ResponseTest
+        fields = [
+            'assessment_id',
+            'name',
+            'description',
+            'subject',
+            'prompt',
+            'sender',
+            'owning_company_id',
+            'owning_company_name',
+        ]
 
 class ResponseTest(AssessmentTool):
```

Comment:
Hello, Arya. Is this response test definition a duplication of the one defined above? If yes, can you please remove them to avoid any confusion during runtime.

Cheers!

Reply:
Will do!